src/index.html

```diff
@@ -3,12 +3,53 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>flexbox</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="logo__img">
+          <a href="#home">
+            <img src="images/logo.png"
+             alt="Moyo logo"
+             >
+          </a>
+        </div>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link  is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
```

Comment:
```suggestion
            </li>
            
            <li class="nav__item">
```

Reply:
<img width="1016" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/7b8929bc-83b8-4d94-8983-ae4a484bdf36">
Fix it in all code